nginx/Dockerfile-prometheus

```diff
@@ -6,7 +6,7 @@ ARG scrapeuri
 ENV SCRAPE_URI $scrapeuri
 
 RUN microdnf install -y git make go
-RUN git clone --branch v0.10.0 https://github.com/nginxinc/nginx-prometheus-exporter
+RUN git clone --branch v0.11.0 https://github.com/nginxinc/nginx-prometheus-exporter
```

Comment:
I updated this as well in my PR that merged earlier: https://github.com/RedHatInsights/turnpike/commit/70d90818b0104fa2661931125d88557a447d1cd5#diff-94e2fe43d93f1dda0f3e42866690289f2f17e08b7d84b6cf387c4b625241e616R8

so I would sync with master and then this should be g2g